tests/monitoring/metrics.go

```diff
@@ -141,19 +122,16 @@ func fetchPrometheusMetrics(virtClient kubecli.KubevirtClient) *libmonitoring.Qu
 	return metrics
 }
 
-func setupVM(virtClient kubecli.KubevirtClient) {
-	vm := createRunningVM(virtClient)
-	libmonitoring.WaitForMetricValue(virtClient, "kubevirt_number_of_vms", 1)
-
-	By("Deleting the VirtualMachine")
-	err := virtClient.VirtualMachine(vm.Namespace).Delete(context.Background(), vm.Name, metav1.DeleteOptions{})
-	Expect(err).ToNot(HaveOccurred())
-
-	libmonitoring.WaitForMetricValue(virtClient, "kubevirt_number_of_vms", -1)
+func basicVMLifecycle(virtClient kubecli.KubevirtClient) {
+	vm := createAndRunVM(virtClient)
+	waitForRunningVMMetrics(virtClient, vm)
+	deleteVM(virtClient, vm)
+	waitForDeletedVMMetrics(virtClient, vm)
 }
```

Comment:
I feel we can remove this func and do it from the BeforeEach(), wdyt?

Reply:
I think in this use case it makes sense to be here. What would be the benefits of moving?

Reply:
Removing one level from the code's depth. This function is used only once and it ONLY calls other functions. Feels redundant.

Reply:
But this is a nit. If you disagree let me know and I'll LGTM.

Reply:
it can be inlined in this function, moving upwards I don't think so

Reply:
but then we would go back to the first implementation